run.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+if [ ! -d "opsfiles" ]; then
+    git clone --depth=1  https://github.com/phanbook/opsfiles.git
+fi
+
+chmod 777 -R public/
+chmod 777 -R content/cache/
+chmod 777 -R content/logs
+
+
+# Now use docker setup for default
+if ! which docker > /dev/null; then
+    echo "Docker doesn't seem to be installed. Please download and install it"
+    echo "from https://docs.docker.com/engine/installation/ and re-run phanbook."
+    exit 3
+fi
+if [[ "destroy" == "$1" ]]; then
+    # Stop and remove any existing containers
+    docker rm -f $(docker ps -a -q)
+    # Delete all images
+    docker rmi -f $(docker images -q)
+fi
+
+if ! which docker-compose > /dev/null; then
+    curl -L https://github.com/docker/compose/releases/download/1.3.3/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+    chmod +x /usr/local/bin/docker-compose
+    sudo service docker start
```

Comment:
Actually there is many commands may needs `sudo`. For example `curl -L ... /usr/local/bin/docker-compose` may not work without rights to write ti the `/usr/local/bin` directory. I propose to remove `sudo` at all. Users will able to run this script as `sudo bash run.sh` if need arises.

Reply:
OK